velox/exec/LocalPlanner.cpp

```diff
@@ -495,7 +497,14 @@ std::shared_ptr<Driver> DriverFactory::createDriver(
     } else if (
         auto windowNode =
             std::dynamic_pointer_cast<const core::WindowNode>(planNode)) {
-      operators.push_back(std::make_unique<Window>(id, ctx.get(), windowNode));
+      if (FLAGS_EnableStreamingWindow) {
```

Comment:
Do not add flags like this. Use PlanNode properties instead.

Reply:
Add isStreaingWindow parameter in WindowNode.